### ISSUE TITLE
WebVTT Validation Requires Hours in Timestamps

### ISSUE THREAD

**Issue by elynema** [samvera-labs/ramp#202]
# Description
The [WebVTT timestamp definition](https://www.w3.org/TR/webvtt1/#webvtt-timestamp) says that hours are optional, if hours are equal to zero. However, the IIIF React Media Player transcript component in Avalon is currently requiring hours, even for media pieces of less than an hour in length. 

We should not require an optional element to be present; the component should allow for all valid timecodes.

# Done Looks Like
- [ ] Ramp Transcript component does not require hours for timecode values

**Comment by masaball** [comment]
Investigation has shown that this had already been fixed on the Ramp side. The validation Regex handles optional hours just fine in my testing. The problem is coming from Avalon still being on IIIF React Media Player 1.2.1. 

I tried for a bit to bump Avalon to the most recent Ramp release and got it to stop producing errors, but no Transcripts at all were loading so there was some other issue occurring. Since bumping us to the newest Ramp doubles up on work @Dananji has to do for #5262, we will be pairing tomorrow (7/25) to walk through the update process and see if we can get transcripts functional again.
